source/instructors/authoring/guides/authoring_guidelines.rst

```diff
@@ -12,7 +12,7 @@ Style your content using Markdown
 You can style your content using Markdown, a simple formatting language that is easier to use than HTML. You can embed HTML snippets in Guide content if you need a feature that is unavailable in Markdown.
 
 
-Load the **Demo Guides and Assessments** project to learn more about authoring with Guides
+Load the Demo Guides and Assessments project to learn more about authoring with Guides
```

Comment:
Wondering why you took the bold off - not a big deal either way

Reply:
Just made the section title on deployed page look weird and looked better without: see https://docs.codio.com/instructors/authoring/guides/authoring_guidelines.html